scripts/filldb.py

```diff
@@ -459,14 +503,20 @@ def createCharts():
     create_last_modification_chart(domain.name)
 
 if __name__ == '__main__':
-        fillPopularityData()
-        fillReleaseFrequencyData()
-        fillBreakingChanges()
-        fillLastModificationDateData()
-        fillLastDiscussedSOData()
-        fillLicenseData()
-        fillIssueClosingTimeData()
-        fillIssueResponseTimeData()
-        fillIssueData()
-        fillOverallScore()
-        createCharts()
+  d = date.today()
+  entrymonth = d.month
+  entryyear = d.year
+  
+  print(entrymonth)
+  
+  fillPopularityData(entrymonth,entryyear)
+  createCharts(entrymonth,entryyear)
+  fillReleaseFrequencyData()
+  fillBreakingChanges()
+  fillLastModificationDateData()
+  fillLastDiscussedSOData()
+  fillLicenseData()
+  fillIssueClosingTimeData()
+  fillIssueResponseTimeData()
+  fillIssueData()
```

Comment:
are these indented correctly?

Reply:
Yes they should be, the code ran fine when I ran it